src/redux/features/participantProjectSpecies/participantProjectSpeciesAsyncThunks.ts

```diff
@@ -0,0 +1,76 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import ParticipantProjectSpeciesService, {
+  AllSpeciesResponse,
+  DeleteProjectSpeciesResponse,
+  ProjectSpecies,
+  ProjectSpeciesResponse,
+} from 'src/services/ParticipantProjectSpeciesService';
+import strings from 'src/strings';
+
+export const requestGetParticipantProjectSpecies = createAsyncThunk(
+  'participantProjectSpecies/get-one',
+  async ({ projectId, projectSpeciesId }: { projectId: number; projectSpeciesId: number }, { rejectWithValue }) => {
+    const response: ProjectSpeciesResponse = await ParticipantProjectSpeciesService.get(projectId, projectSpeciesId);
+
+    if (response && response.requestSucceeded) {
+      return response.data.projectSpecies;
+    }
+
+    return rejectWithValue(strings.GENERIC_ERROR);
+  }
+);
+
+export const requestListParticipantProjectSpecies = createAsyncThunk(
+  'participantProjectSpecies/list',
+  async (projectId: number, { rejectWithValue }) => {
```

Comment:
We need a requestListParticipantProjectSpecies for all projects (for the species list view), not just for a specific project, maybe this projectId could be optional?

Reply:
Oh, never mind. I just realize I can use the getDeliverables and filter the result by type

Reply:
@nickgraz I know this wasn’t in the Jira ticket, buy maybe we missed it?

Reply:
That wasn't in the AC in the ticket (the ticket actually had great AC this time, thanks @nickgraz), so that's why it's not included in the PR. But I can definitely add that if we need it, no problem.

Great suggestion @constanzauanini! I'll give that a try.

Reply:
This request is used to show all of the participant project species that have already been associated. They will be shown within the "species deliverable" view. A project ID must be required here because it is the basis of the relationship between the participant project and the organization species. 

We already have selectors elsewhere for "show all species for an organization", which will be used in the "add species to my participant project" flow.

Reply:
Ok that makes sense, thanks for the context. Will revert the optional `projectId` param changes 👍

Reply:
Thanks @nickgraz for context, so then in the "search all species for an organization" we need to add a field to get the projects that species is part of? Could we mock that too?

Reply:
I believe that feature (projects within species) is only available to participants, so we will just combine the two lists. If they are a participant, get the org species and participant project species. If they aren't, only fetch the org species and don't mention "project" anywhere. I will double check this assumption

Reply:
I thought same thing, that's why I think we need a "get all participant project species" selector

Reply:
I believe that is what this selector does `selectParticipantProjectSpeciesListRequest`

Reply:
Correct, but what would be the corresponding thunk to dispatch? We need a projectId to use this one

Reply:
Oh, I see what you're saying. Since there is only a single argument being passed (projectId) we can replace the request ID with it by adding `true` to the `buildReducers` call in the slice that uses this thunk. Good catch!

Reply:
https://github.com/terraware/terraware-web/pull/2567

Reply:
Thanks for the PR with the change, but we still need to get all project species, without a projectId. I made a comment in the other PR, I think the selectParticipantProjectSpeciesListRequest needs a requestId, but not a projectId, and the selectParticipantProjectSpeciesRequest needs a projectId. But do we need to change the thunk to allow not having a projectId?

Reply:
Why do we need to "get all project species" without a project ID? Project species only exist in the context of a participant project, and the only access patterns for getting all project species is in the context of a specific participant project. 

Also, the "participant project species" model will have `id, project_id, species_id, status, feedback, rationale`. So we actually don't need `project_id` to "get single participant project species" since the `id` of the row is sufficient to retrieve all the data for that single participant project species.

Reply:
I need to remove `projectId` from `requestGetParticipantProjectSpecies`, it should only need the participant project species ID.

Reply:
Oh, I see. Yea I think we will be able to add that in the search API request. The species stable uses the Search API to show the rows, and we will add `participant_project_species` as a Search API sublist, so all we will need to do is add a search field like `participantProjectSpecies.projects.name`.

Reply:
Thanks! Sorry for all the confusion!

Reply:
No problem! 🤜 🤛